kokoro/linux/java_aarch64/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# This is the top-level script we give to Kokoro as the entry point for
+# running the "continuous" and "presubmit" jobs.
+
+set -ex
+
+# Change to repo root
+cd $(dirname $0)/../../..
```

Comment:
cd $(git rev-parse --show-toplevel)

seems more reliable than counting directories

Reply:
Let's leave as is as this is the "standard" way it's currently done in most protobuf's CI testing scripts.
I think it's better to stay consistent with the way it's done in most places in this repository.

Examples:
https://github.com/protocolbuffers/protobuf/blob/3f5fc4df1de8e12b2235c3006593e22d6993c3f5/kokoro/release/python/linux/build_artifacts.sh
https://github.com/protocolbuffers/protobuf/blob/3f5fc4df1de8e12b2235c3006593e22d6993c3f5/kokoro/release/protoc/linux/build.sh
https://github.com/protocolbuffers/protobuf/blob/3f5fc4df1de8e12b2235c3006593e22d6993c3f5/kokoro/linux/benchmark/run.sh